tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -36,9 +31,95 @@ In most scenarios, you are recommended to use private endpoint connection over V
+
+    ![Verify endpoint service](/media/tidb-cloud/private-endpoint/create-endpoint-2.png)
+
+3. Select **Other endpoint services**.
+4. Enter the **Service Name**.
+5. Click **Verify service**.
+6. Select your VPC in the drop-down list. Click on **Additional settings** icon and select the **Enable DNS name** checkbox.
+7. Select the availability zone where your TiDB cluster is located in the **Subnets** area.
+8. Select your security group properly in the **Security groups** area.
+
+    > **Note:**
+    >
+    >  Make sure the selected security group allows inbound access from your EC2 instances on Port 4000 or a customer-defined port.
+
+9. Click **Create endpoint**.
 
-In addition to the [prerequisites](#prerequisites), there are 5 steps to set up a private endpoint connection with AWS PrivateLink:
+</div>
+<div label="Use AWS CLI">
+
+To use the AWS CLI to create a VPC interface endpoint, perform the following steps:
+
+1. To get the **VPC ID** and **Subnet ID**, navigate to your AWS Management Console, and locate them in the relevant sections. Ensure that you find the **Subnet ID** within the **Availability Zone ID** that you found in [step 1](#step-1-choose-a-tidb-cluster).
+2. Copy the command provided below and replace the relevant arguments with the information you obtained, then execute it in your terminal.
+
+```bash
+aws ec2 create-vpc-endpoint --vpc-id ${your_vpc_id} --region ${region_id} --service-name ${service_name} --vpc-endpoint-type Interface --subnet-ids ${your_subnet_id}
+```
+
+> **Tip:**
+>
+> - Before running the command, you need to have AWS CLI installed and configured. See [AWS CLI configuration basics](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) for details.
+
+</div>
+</SimpleTab>
+
+Then you can connect to the endpoint service with the private DNS name.
+
+#### Step 3: Connect to your TiDB cluster
+
+After you have created the interface endpoint, go back to the TiDB Cloud console and take the following steps:
+
+1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target cluster to go to its overview page.
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+3. Select the **Private Endpoint** tab.
+4. Under **Connect With**, click the tab of your preferred connection method, and then connect to your cluster with the connection string.
+
+> **Tip:**
+>
+> If you cannot connect to the cluster, the reason might be that the security group of your VPC endpoint in AWS is not properly set. See [this FAQ](#troubleshooting) for solutions.
+>
+> If you encounter an error while creating a VPC endpoint that reads "private-dns-enabled cannot be set because there is already a conflicting DNS domain for gatewayXX-privatelink.XX.prod.aws.tidbcloud.com in the VPC vpc-XXXXX," it is due to the fact that a private endpoint has already been created, and creating a new one is unnecessary.
```

Comment:
```suggestion
> When creating a VPC endpoint, if you encounter an error `private-dns-enabled cannot be set because there is already a conflicting DNS domain for gatewayXX-privatelink.XX.prod.aws.tidbcloud.com in the VPC vpc-XXXXX`, it is due to that a private endpoint has already been created, and creating a new one is unnecessary.
```